pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["pip", "setuptools>=64"]
```

Comment:
`pip` shouldn't be a build-system requirement. Only `setuptools` provides the build backend.

Reply:
setup tools is necessary to use this tool and it was also a hidden dependency, see comments here:
https://github.com/fake-useragent/fake-useragent/pull/430#issuecomment-2576009045

Agree that it is not necessary to use as build system, it was a refuse not removed